### ISSUE TITLE
Add Projects list 

### ISSUE THREAD

**Issue by Franklin89** [Franklin89/Blog#44]

**Comment by thomascayne** [comment]
@Franklin89 Yes, please. Add Projects files/list. I found your `adding-health-checks-to-asp-net-core` through Google search because I am adding health checks to my web api. There's no implement of `IsApplicationHealthy()`. So what do it do?
